src/main/java/org/team199/robot2020/RobotContainer.java

```diff
@@ -58,13 +62,16 @@
     private final Climber climber = new Climber();
     private final RobotPath[] paths;
     private final LinearInterpolation linearInterpol;
+    private final UsbCamera camera1 = MotorControllerFactory.configureCamera(Constants.Drive.kCamera1Port);
+    private final VideoSink cameraServer = CameraServer.getInstance().getServer();
 
     public RobotContainer() {
         
         if(DriverStation.getInstance().getJoystickName(0).length() != 0) {
             configureButtonBindingsLeftJoy();
         } else{
             System.err.println("ERROR: Dude, you're missing the left joystick.");
+            ToggleCamera.setCamera(0);
```

Comment:
Setting the default camera should happen in the subsystem that manages the cameras.